typings/index.d.ts

```diff
@@ -2184,6 +2207,95 @@ declare module 'discord.js' {
 
   type APIMessageContentResolvable = string | number | boolean | bigint | symbol | readonly StringResolvable[];
 
+  interface APIRawMessage {
```

Comment:
This interface is incomplete.. why don't we use `discord-api-types`? I also don't remember the discussion for that

Reply:
I'm happy to use it, just not sure if this PR should be covering that. I copied the interface from the PR izeki referenced in his comments.

Reply:
It should use `discord-api-types`, we're within semver-major (v13) changes, and we'll eventually use the library for REST's routes.